tuxemon/monster.py

```diff
@@ -297,6 +303,8 @@ def load_from_db(self, slug):
                 self.type2 = results["types"][1].lower()
 
         self.weight = results["weight"]
+        self.catch_rate = results["catch_rate"]
```

Comment:
The correct way to handle this would be to use 
results.get("catch_rate", TuxemonConfig().default_monster_catch_rate)

This ensures that we get the default when a record is missing the catch_rate instead of just crashing.

Reply:
I have implemented the change you proposed. Thank you for pointing it out!